### PR TITLE
Xbouma/user tests

### PR DESCRIPTION
**Pull request by tdtom167**

### CODE REVIEW

File: rocket-mission-api/src/main/java/cz/muni/fi/facade/RocketFacade.java [hunk 1]
```diff
@@ -37,7 +38,7 @@
      *
      * @param rocket rocket to update
      */
-    void updateRocket(RocketDTO rocket);
+    void updateRocket(UpdateRocketDTO rocket);
```
Comment (xMartinezz) [inline rocket-mission-api/src/main/java/cz/muni/fi/facade/RocketFacade.java:41]: return value should be RocketDTO

File: rocket-mission-persistence/src/main/java/cz/muni/fi/dao/MissionDao.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
      *
      * @param mission mission to add
      */
-    void addMission(Mission mission);
```
Comment (xMartinezz) [inline rocket-mission-persistence/src/main/java/cz/muni/fi/dao/MissionDao.java:19]: javadocs for return value missing

File: rocket-mission-service/src/test/java/cz/muni/fi/services/MissionServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import cz.muni.fi.entity.Mission;
+
+public class MissionServiceImplTest {
+
+    static Mission createMission(String name) {
+        Mission mission = new Mission();
+        mission.setName(name);
+        return mission;
+    }
+}
```
Comment (xkostka2) [inline rocket-mission-service/src/test/java/cz/muni/fi/services/MissionServiceImplTest.java:12]: This shouldn't be part of this PR
